Actions/AL-Go-Helper.ps1

```diff
@@ -369,20 +369,21 @@ function ReadSettings {
         "keyVaultClientIdSecretName"             = ""
         "codeSignCertificateUrlSecretName"       = "CodeSignCertificateUrl"
         "codeSignCertificatePasswordSecretName"  = "CodeSignCertificatePassword"
-        "supportedCountries"                     = @()
+        "additionalCountries"                    = @()
         "appDependencies"                        = @()
         "appFolders"                             = @()
         "testDependencies"                       = @()
         "testFolders"                            = @()
         "installApps"                            = @()
         "installTestApps"                        = @()
-        "applicationDependency"                  = "19.0.0.0"
+        "applicationDependency"                  = "18.0.0.0"
```

Comment:
is it intentional ?

Reply:
Yes, this is the lowest supported version - 19 was a bit harsh.
scripts will enumerate app.jsons to find the actual dependencies - if they are lower than 18.0 - it will fail.